modules/document_repository/ajax/addCategory.php

```diff
@@ -61,7 +63,7 @@
     $www = $config->getSetting('www');
 
     $msg_data['newCategory'] = $www['url'] . 
-                               "/main.php?test_name=document_repository";
+                              $baseURL . "/document_repository/";
```

Comment:
$www['url'] and $baseURL have the same value. There will be a duplication of hostname.